flink-table/flink-sql-gateway-api/src/main/java/org/apache/flink/table/gateway/api/results/ResultSet.java

```diff
@@ -23,16 +23,30 @@
 import org.apache.flink.table.catalog.ResolvedSchema;
 import org.apache.flink.table.data.RowData;
 
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
 import javax.annotation.Nullable;
 
+import java.io.Serializable;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 /** The collection of the results. */
 @PublicEvolving
-public class ResultSet {
+@JsonSerialize(using = ResultSetJsonSerializer.class)
+@JsonDeserialize(using = ResultSetJsonDeserializer.class)
+public class ResultSet implements Serializable {
```

Comment:
It seems this class needs a serial version id

Reply:
Fixed.